### PR TITLE
Kkraune/visiting

### PR DESCRIPTION
**Pull request by kkraune**
maybe easier to review per commit

### CODE REVIEW

File: en/content/visiting.html [hunk 1]
```diff
@@ -83,26 +83,16 @@ <h2 id="fields">Fields</h2>
...
-Export data to stdout:
-<pre>
-$ vespa-visit
-</pre>
+Export data to stdout.
 As a full data dump takes time / space / resources, refine as needed:
 <pre>
-$ vespa-visit -v -p progress-file -s 'id.hash().abs() % 100 = 0' \
-  --fieldset 'music:[document]' > 100.json
+$ vespa-visit -v -p progress-file -s 'id.hash().abs() % 100 = 0' > 100.json
```
Comment (vekterli) [inline en/content/visiting.html:91]: Should we replace this with a `/document/v1` example that uses slicing instead? The `id.hash()` construction requires a complete iteration, whereas slicing inherently limits to a subset of the data space. However, slicing is currently not supported by the `vespa-visit` CLI.

File: en/content/visiting.html [hunk 2]
```diff
@@ -491,3 +490,21 @@ <h2 id="visitor-targets">Visitor targets</h2>
     </tr>
   </tbody>
 </table>
+
+
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+<p>
+  Normally all documents share the same <a href="buckets.html#bucket-space">bucket space</a>
+  - documents for multiple schemas are co-located.
```
Comment (vekterli) [inline en/content/visiting.html:499]: Typography nerd nitpick: consider replacing with non-spaced em-dash 👀 (macOS shortcut: `shift` `option` `-`)
```suggestion
  Normally all documents share the same <a href="buckets.html#bucket-space">bucket space</a>—documents
  for multiple schemas are co-located.
```

File: en/content/visiting.html [hunk 3]
```diff
@@ -491,3 +490,21 @@ <h2 id="visitor-targets">Visitor targets</h2>
...
+  When using <a href="../parent-child.html">parent/child</a>, global documents are stored in a separate bucket space.
+  Use the <a href="../reference/document-v1-api-reference.html#bucketspace">bucketSpace</a> parameter
+  to visit the <code>default</code> or <code>global</code> space.
+  This is a common problem when dumping all documents and dumped count is not the expected count.
+</p>
+<p>
+  A visit operation might stall/hang if the content cluster is in an inconsistent state -
+  replicas are still mering between nodes.
```
Comment (vekterli) [inline en/content/visiting.html:507]: Nit: typo. Also smuggling in a sneaky em dash 🎶🎺
```suggestion
  A visit operation might stall/hang if the content cluster is in an inconsistent
  state—replicas are still merging between nodes.
```

**Review by vekterli — APPROVED** [review]
:+1:

File: en/content/visiting.html (hunk 2 above)
Comment (kkraune) [reply en/content/visiting.html:499]: I saw that coming :-)

File: en/content/visiting.html (hunk 1 above)
Comment (kkraune) [reply en/content/visiting.html:91]: That is a great suggestion! I punt this to the next PR, adding this as a ToDo, as I hope we can get the vespa-cli to support visiting and might rewrite examples at a larger scale.

**Pull request merged**